app/models/content_store_payload_version.rb

```diff
@@ -0,0 +1,41 @@
+class ContentStorePayloadVersion < ActiveRecord::Base
+  def self.increment(content_item_id:)
```

Comment:
Both of those examples take multiple options. The method in question takes a single named argument.